src/instructlab/config/init.py

```diff
@@ -109,7 +109,6 @@ def init(
     cfg.serve.model_path = model_path
     cfg.generate.taxonomy_path = taxonomy_path
     cfg.generate.taxonomy_base = taxonomy_base
-    cfg.evaluate.model = model_path
```

Comment:
is it intentional?

Reply:
yes, it does not make sense for eval to have a default given that you evaluate the model given by training

Reply:
OK. This goes in the other direction from the other patch [adding](https://github.com/instructlab/instructlab/pull/1897) teacher model path though, so the two changes will have to reconcile somehow. I understand there are opinions about `--model-path` option (e.g. that it should be [removed](https://github.com/instructlab/instructlab/issues/1976).) but is it something this PR **has** to touch now?